atmosphere/settings/__init__.py

```diff
@@ -41,16 +41,14 @@
 ALLOWED_HOSTS = [unicode(SERVER_URL.replace('https://', ''))]
 
 # NOTE: first admin will be sender of atmo emails.
-ADMINS = (
-    ('AT LEAST ONE ADMIN REQUIRED', 'sends-email@if-debug-false.com'),
-)
+ADMIN = ('AT LEAST ONE ADMIN REQUIRED', 'sends-email@if-debug-false.com')
```

Comment:
Comment on 43 can be nuked, and first value of dummy ADMIN no longer makes sense